backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -305,7 +306,7 @@ export class UserResolver {
   @Authorized([RIGHTS.CREATE_USER])
   @Mutation(() => User)
   async createUser(
-    @Args() { email, firstName, lastName, language, publisherId }: CreateUserArgs,
+    @Args() { email, firstName, lastName, language, publisherId, redeemCode }: CreateUserArgs,
```

Comment:
```suggestion
    @Args() { email, firstName, lastName, language, publisherId, redeemCode = null}: CreateUserArgs,
```

Do we need a default value here?